pyaedt/application/Analysis3DLayout.py

```diff
@@ -34,33 +65,45 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def oboundary(self):
-        """:return: BoundarySetup Module object"""
+        """Boundary object."""
         return self._odesign.GetModule("Excitations")
 
     @property
     def mesh(self):
-        """ """
+        """Mesh object."""
         return self._mesh
 
     @property
     def get_excitations_name(self):
-        """:return: BoundarySetup Module object"""
+        """Excitation names.
+        
+        Returns
+        -------
+        type
+            BoundarySetup module object.
+        
+        """
         return list(self.oboundary.GetAllPortsList())
 
     @property
     def get_all_sparameter_list(self, excitation_names=[]):
-        """Get the list of all the SParameter from a list of exctitations. If no excitation is provided it will provide a full list of sparameters
-        Example: excitation_names ["1","2"] output ["S(1,1)", "S(1,2)", S(2,2)]
+        """List of all S parameters for a list of excitations. 
 
         Parameters
         ----------
-        excitation_names :
-            list of excitation to include (Default value = [])
-
+        excitation_names : list, optional
+            List of excitations. The default is ``[]``, in which case
+            the S parameters for all excitations are to be provided. 
+            
         Returns
         -------
-        type
-            list of strin representing Sparameters of excitations
+        list
+            List of strings representing the S parameters of the excitations.
+            
+        Examples
+        --------
+        
+        >>>excitation_names ["1","2"] output ["S(1,1)", "S(1,2)", S(2,2)]
 
```

Comment:
```suggestion
```
Let's just remove this.  Examples should have a full example in Python like:
```
>>> aedtapp.get_all_sparameter_list(...)
```
I'm not adding one here right now since I'm not 100% sure of the format and would like someone to actually run this and place the return in the output.